### PR TITLE
Feature #619 - Documentation logic

### LINKED ISSUE #619: Update documentation for logic
**Tasks**

- [ ] Add (in Dutch) example form to show some logic rules in a form (verhuizen?). See [here](https://open-forms.readthedocs.io/en/latest/manual/forms/examples/autofill_address.html) for example)
- [ ] Add (in Dutch) explaining logic in general

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #619

### CODE REVIEW

File: docs/manual/forms/basics.rst [hunk 1]
```diff
@@ -105,48 +105,83 @@ Stappen en velden
...
+^^^^^^
+
+Nadat een regel is gedefinieerd, kunnen een of meerdere acties worden toegevoegd. De acties kunnen het formulier
+wijzigen afhankelijk van de gegevens die de gebruiker invult. Er zijn verschillende acties mogelijk:
+
+* Blokkeer doorgaan naar de volgende stap.
+
+* Wijzig een attribuut van een veld/component. Met dit actie kan:
+
+    * Een niet-verplicht veld worden veranderd in een verplicht veld (of andersom)
```
Comment (joeribekker) [inline docs/manual/forms/basics.rst:179]:
```suggestion
    * Een niet-verplicht veld wordt veranderd in een verplicht veld (of andersom)
```

Het wordt; Zij worden

File: docs/manual/forms/basics.rst [hunk 2]
```diff
@@ -105,48 +105,83 @@ Stappen en velden
...
+
+    * Een zichtbaar veld worden verborgen (of een verborgen veld zichtbaar worden)
```
Comment (joeribekker) [inline docs/manual/forms/basics.rst:181]:
```suggestion
    * Een zichtbaar veld wordt verborgen (of een verborgen veld wordt zichtbaar)
```

File: docs/manual/forms/basics.rst [hunk 3]
```diff
@@ -105,48 +105,83 @@ Stappen en velden
...
+
+    * Een veld kan worden uitgeschakeld
+
+* Wijzig de waarde van een veld/component aan. Met dit actie kan de waarde van een veld/component worden veranderd naar de waarde van een andere component of een waarde gekozen door de beheerder.
```
Comment (joeribekker) [inline docs/manual/forms/basics.rst:185]:
```suggestion
* Wijzig de waarde van een veld/component. Met deze actie kan de waarde van een veld/component worden veranderd naar de waarde van een andere component of een waarde gekozen door de beheerder.
```

File: docs/manual/forms/basics.rst [hunk 4]
```diff
@@ -105,48 +105,83 @@ Stappen en velden
...
+
+* Markeer een stap als 'niet van toepassing' (n.v.t).
```
Comment (joeribekker) [inline docs/manual/forms/basics.rst:187]: Can you add a screenshot that shows a few different logic rules?

File: docs/manual/forms/examples/logic_rules.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+20. Selecteer de component **Warning** in de eerst drop down, dan **verborgen** en **Nee** in de laatste drop-down.
+
+.. note::
+
+    Deze twee acties betekenen: als de gebruiker jonger dan 18 is, dan blokkeer
+    doorgaan naar de volgende formulier stap en maak de 'Warning component' zichtbaar.
+
+21. Klik onderaan op **Opslaan** om het formulier volledig op te slaan.
+
+U kunt nu het formulier bekijken.
```
Comment (joeribekker) [inline docs/manual/forms/examples/logic_rules.rst:75]: Can you add a screenshot here that shows the created logic rules?

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: docs/manual/forms/basics.rst (hunk 1 above)
Comment (SilviaAmAm) [reply docs/manual/forms/basics.rst:179]: wooops :sweat_smile:

**Review by joeribekker — APPROVED** [review]

**Pull request merged**